编译错误

# 运行命令：
```shell
bazel build //... -c dbg --config=macos-asan
```
# 报错日志
```markdown
Loading: 
Loading: 0 packages loaded
Analyzing: 184 targets (0 packages loaded, 0 targets configured)
DEBUG: Rule 'org_interconnection' indicated that a canonical reproducible form can be obtained by modifying arguments shallow_since = "1669271462 +0800"
DEBUG: Repository org_interconnection instantiated at:
  /Users/dsy/Desktop/yacl/WORKSPACE:19:10: in <toplevel>
  /Users/dsy/Desktop/yacl/bazel/repositories.bzl:59:10: in yacl_deps
  /private/var/tmp/_bazel_dsy/4f9993a0cd77836fcc6fa6eaf86db19c/external/bazel_tools/tools/build_defs/repo/utils.bzl:233:18: in maybe
Repository rule git_repository defined at:
  /private/var/tmp/_bazel_dsy/4f9993a0cd77836fcc6fa6eaf86db19c/external/bazel_tools/tools/build_defs/repo/git.bzl:199:33: in <toplevel>
DEBUG: Rule 'simplest_ot' indicated that a canonical reproducible form can be obtained by modifying arguments shallow_since = "1658802777 +0800"
DEBUG: Repository simplest_ot instantiated at:
  /Users/dsy/Desktop/yacl/WORKSPACE:19:10: in <toplevel>
  /Users/dsy/Desktop/yacl/bazel/repositories.bzl:51:10: in yacl_deps
  /private/var/tmp/_bazel_dsy/4f9993a0cd77836fcc6fa6eaf86db19c/external/bazel_tools/tools/build_defs/repo/utils.bzl:233:18: in maybe
Repository rule git_repository defined at:
  /private/var/tmp/_bazel_dsy/4f9993a0cd77836fcc6fa6eaf86db19c/external/bazel_tools/tools/build_defs/repo/git.bzl:199:33: in <toplevel>
INFO: Analyzed 184 targets (0 packages loaded, 0 targets configured).
INFO: Found 184 targets...
[0 / 18] [Prepa] Creating source manifest for //yacl/crypto/base/hash:ssl_hash_all_test
[26 / 45] Compiling yacl/crypto/base/hash/ssl_hash_all_test.cc; 1s darwin-sandbox ... (8 actions, 7 running)
[32 / 45] Compiling yacl/crypto/base/aead/sm4_mac.cc; 2s darwin-sandbox ... (8 actions, 7 running)
ERROR: /Users/dsy/Desktop/yacl/yacl/crypto/base/ecc/BUILD.bazel:27:16: Compiling yacl/crypto/base/ecc/ecc_spi.cc failed: (Aborted): wrapped_clang_pp failed: error executing command external/local_config_cc/wrapped_clang_pp -fstack-protector -fcolor-diagnostics -Wall -Wthread-safety -Wself-assign -fno-omit-frame-pointer -g '-std=c++11' 'DEBUG_PREFIX_MAP_PWD=.' -iquote . -iquote ... (remaining 55 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
yacl/crypto/base/ecc/ecc_spi.cc:95:105: error: use of undeclared identifier 'ex'
      SPDLOG_DEBUG("Ec lib {} do not support curve {}, msg={}, try next ...",perf_item.second, ec_name, ex.what());
                                                                                                        ^
1 error generated.
Error in child process '/usr/bin/xcrun'. 1
INFO: Elapsed time: 2.948s, Critical Path: 2.55s
INFO: 43 processes: 29 internal, 14 darwin-sandbox.
FAILED: Build did NOT complete successfully
FAILED: Build did NOT complete successfully
```

# 分析
```markdown
问题出在“yacl/crypto/base/ecc/ecc_spi.cc:95:105: error: use of undeclared identifier 'ex'
      SPDLOG_DEBUG("Ec lib {} do not support curve {}, msg={}, try next ...",perf_item.second, ec_name, ex.what());”

当我修改这行代码为“SPDLOG_DEBUG("Ec lib {} do not support curve {}, msg=, try next ...",perf_item.second, ec_name);”
**可以正常通过**，这只是临时解决方法，请yacl团队评估。
```

--- a/yacl/crypto/base/ecc/ecc_spi.cc
+++ b/yacl/crypto/base/ecc/ecc_spi.cc
@@ -92,8 +92,8 @@ std::unique_ptr<EcGroup> EcGroupFactory::Create(const CurveName& ec_name) {
   auto meta = GetCurveMetaByName(ec_name);
   for (const auto& perf_item : GStore::PerformanceMap()) {
     if (!GStore::CheckerMap().at(perf_item.second)(meta)) {
-      SPDLOG_DEBUG("Ec lib {} do not support curve {}, msg={}, try next ...",
-                   perf_item.second, ec_name, ex.what());
+      SPDLOG_DEBUG("Ec lib {} do not support curve {}, try next ...",
+                   perf_item.second, ec_name);
       continue;
     }
     return GStore::CreatorMap().at(perf_item.second)(meta);

